WizardWorldApi/WizardWorld.Persistance/Configurations/HouseConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using WizardWorld.Persistance.Models.Houses;
+
+namespace WizardWorld.Persistance.Configurations {
+    public class HouseConfiguration : IEntityTypeConfiguration<House> {
+        public void Configure(EntityTypeBuilder<House> builder) {
+            builder
```

Comment:
You need to add one-to-many relation here. 

https://www.learnentityframeworkcore.com/configuration/one-to-many-relationship-configuration